build/vsts_build.yaml

```diff
@@ -862,11 +862,6 @@ jobs:
       artifactName: "$(VsixPublishDir)"
       downloadPath: "$(Build.Repository.LocalPath)/artifacts"
 
-  - task: NuGetToolInstaller@0
-    displayName: "Use NuGet 4.5.0"
```

Comment:
Is this delete needed as part of the fix?

Reply:
It's not a part of the fix. But it failed one of my builds at one point, so I removed it since it wasn't necessary anyways.